package.json

```diff
@@ -180,7 +172,6 @@
     "deep-freeze": "0.0.1",
     "dice-coefficient": "^2.1.0",
     "distance-transform": "^1.0.2",
-    "eslint-plugin-ava": "^13.2.0",
```

Comment:
There's probably no replacement for the ava https://github.com/avajs/eslint-plugin-ava/blob/main/docs/rules/no-only-test.md, right? I remember that we added this one just recently, because we got bitten by that.

Reply:
Good point! I replaced it with a very simplistic bash file: https://github.com/scalableminds/webknossos/pull/6618/commits/11fbb9292c90cd5c60b96643325f297dd7dcd0e6
It's not super accurate (since the only modifier could be formulated in another way), but I think it should catch 95% of the cases :)

Reply:
Sweet, thank you!